src/Core/DefaultGeometryLayers.js

```diff
@@ -93,7 +93,21 @@ function _tiledPreUpdate(preUpdateSpecialisation) {
     };
 }
 
-export function createGlobe(id, options) {
+/**
+ * Creates a Globe geometry, which can be used to draw other layers (color, elevation, ...)
```

Comment:
"Creates a **g**lobe geometry, **on** which other layers (color, elevation, ...) **can be drawn**"